docs/explanation/parts.rst

```diff
@@ -0,0 +1,263 @@
+Parts
+=====
+
+In the Craft Parts framework, parts are descriptions of the components to be
+build for deployment, either individually or as part of a larger build
+containing many components.
+
+Tools like `Snapcraft`_ and `Juju`_ that use the concepts of parts to
+describe a build process typically accept specifications of parts in YAML format. This allows each part to be described in a convenient,
+mostly-declarative format.
+
+When the Craft Parts framework is used to process a part on behalf of a tool
+or library, it performs the steps described in the
+:ref:`parts lifecycle <lifecycle>` to *pull* the source code and dependencies, *overlay* them onto a base file system, *build* the part, then *organize*,
```

Comment:
The lifecycle steps are pull, overlay, build, stage, and prime. "Organize" is an operation performed during the build step and not a separate step in the lifecycle.

Reply:
Thanks. I've moved "organize" into the build section for the next draft.